media-plugins/gst-plugins-qt5/gst-plugins-qt5-1.18.4.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+GST_ORG_MODULE=gst-plugins-good
+
+inherit gstreamer-meson
+
+DESCRIPTION="Qt5 QML video sink plugin"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+IUSE="+egl wayland +X orc"
+
+REQUIRED_USE="
+	wayland? ( egl )
+"
+
+GST_REQ="${PV}"
```

Comment:
Not really sure why you would define this variable.

Reply:
I have no idea, I copied that from another gstreamer ebuild. :D

Reply:
This was used in the dependency below, but I could just replace it by using the PV there directly.